packages/ui/src/components/ListItem/ContactListItem.tsx

```diff
@@ -17,6 +18,7 @@ export const ContactListItem = ({
   showEndContent = false,
   endContent,
   matchText,
+  subtitle,
```

Comment:
Not critical, but I think more conventional in our codebase would be a `showStatus` property?

Reply:
If it were clear it would always be used that way, I'd agree. _ContactListItem_ is sufficiently broad that I'd rather leave it generic for now